src/platforms/unity/configuration/il2cpp.mdx

```diff
@@ -4,11 +4,7 @@ description: "Learn how the SDK provides line numbers to issues from IL2CPP buil
 sidebar_order: 10
 ---
 
-<Note>
-
-This feature is available from Unity versions `2020.3` and `2021.3` onwards. We are running tests in CI against the current LTS, starting from `2020.3.37f1` and `2021.3.6f1`.
-
-</Note>
+Sentry can give you line numbers on C# exceptions in **release builds** with IL2CPP, on Unity versions `2020.3` and `2021.3` and later.
```

Comment:
```suggestion
Sentry can provide you with line numbers on C# exceptions in **release builds** with IL2CPP, on Unity versions `2020.3`, `2021.3`, and later.
```

Reply:
Made the change separately due to a conflict in applying both suggestions at once